scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -547,6 +538,25 @@ synchronized void checkChangeJobPriority(JobId jobId, JobPriority priority) thro
         }
     }
 
+    synchronized void checkPermissions(String methodName, IdentifiedJob identifiedJob, String errorMessage)
+            throws NotConnectedException, UnknownJobException, PermissionException {
+        try {
+            checkJobOwner(methodName, identifiedJob, errorMessage);
```

Comment:
This method should throw an exception, like it was before. so all the existing code will not be changed.